kie-pmml-trusty/kie-pmml-evaluator/kie-pmml-evaluator-core/src/main/java/org/kie/pmml/evaluator/core/utils/PostProcess.java

```diff
@@ -141,6 +142,15 @@ static void populateOutputFields(final PMML4Result toUpdate,
                                                                           toUpdate,
                                                                           processingDTO));
         }
+
+        populateOutputFields(toUpdate, processingDTO, outputFieldsByFeature.get(RESULT_FEATURE.PREDICTED_DISPLAY_VALUE), model::getPredictedDisplayValue);
```

Comment:
Hi, I would prefer to have consistency on similar code. In this method there are three kinds of OutputFields written/managed in one way, and the others (the ones you add) managed in different way. This makes the code hard to maintain and understand.
For the moment being I think it is better to write all of them the same way - later on we may eventually think to a better abstraction. WDYT ?